src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesFactoryAdapter.java

```diff
@@ -171,6 +173,11 @@ public KubernetesClient createClient() throws NoSuchAlgorithmException, Unrecove
         builder = builder.withRequestTimeout(readTimeout * 1000).withConnectionTimeout(connectTimeout * 1000);
         builder.withMaxConcurrentRequestsPerHost(maxRequestsPerHost);
 
+        if(httpsProxy != null && !httpsProxy.isEmpty()) {
```

Comment:
```suggestion
        if(httpsProxy != null) {
```
It's already checked above